geode-core/src/distributedTest/java/org/apache/geode/management/internal/cli/NetstatDUnitTest.java

```diff
@@ -71,7 +73,12 @@ public static void beforeClass() throws Exception {
     gfsh.connectAndVerify(locator0);
 
     netStatLsofCommand = "netstat --with-lsof=true --member=" + server1.getName();
+  }
 
+  @Before
+  public void postSetup() {
+    // Add ignored exceptions to ignore IllegalArgumentException from MemoryUsage java obj
+    IgnoredException.addIgnoredException("committed = 538968064 should be < max = 536870912");
```

Comment:
I think it would be better to change the product to handle this jdk bug (see: https://bugs.openjdk.java.net/browse/JDK-8207200). Kirk had an idea of how to do that. Basically when VMStats50.refresh calls getHeapMemoryUsage catch IllegalArgumentException, log something (once?), and then do the remainder of the refresh method. This prevents us from updating some stats but at least by catching the exception we would do the other stats. A user might use this jdk so it seems like it is worth us handling this unexpected exception.